test/test_maxwellDriver1d.py

```diff
@@ -1057,3 +1058,64 @@ def test_buildDrivedEvolutionOperator():
     plt.spy(A_drived, markersize=4)
     # plt.spy(A, markersize=4)
     plt.show()
+
+def test_check_initial_conditions_GW_right():
+
+    x_min = -4.0
+    x_max = 4.0
+    k_elements = 400
+
+    sp = FDTD1D(mesh=Mesh1D(x_min, x_max, k_elements, boundary_label="PEC"))
+    driver = MaxwellDriver(sp, timeIntegratorType='LF2', CFL=1.0)
+
+    s0 = 0.25
+    initialFieldE = np.exp(-(sp.x)**2/(2*s0**2))
+    initialFieldH = np.exp(-(sp.xH)**2/(2*s0**2))
+
+    d = np.diff(sp.x)[0]
+    n_sigmas=3
+    
+    driver['E'][:] = initialFieldE[:]
+    driver['H'][:] = initialFieldH[:]
+
+    x_center = k_elements / 2
+    one_time_unit_elements = k_elements / (x_max - x_min)
+
+    E_gaussian_start = driver['E'][int(x_center - n_sigmas*s0 / d)\
```

Comment:
The final field would be something like in the whole domain if the wave has not bounced anywhere (you could use periodic conditions to avoid this).
`finalFieldE = np.exp(-(sp.x - distance_traveled)**2/(2*s0**2))`